appveyor.yml

```diff
@@ -2,12 +2,23 @@ version: 1.0.{build}
 image: Visual Studio 2017
 build_script:
 - ps: .\build.ps1
+
 test:
   assemblies:
     only:
     - '**\*tests.dll'
+
 artifacts:
 - path: .\code_drop\packages\*.nupkg
   name: Nuget packages
 - path: .\code_drop\log\*
-  name: Logs
\ No newline at end of file
+  name: Logs
+
+deploy:
+  provider: NuGet
+  server:  https://www.myget.org/F/roundhouse/api/v2/package
+  api_key:
+    secure: <add MyGet api key encrypted with https://ci.appveyor.com/tools/encrypt>

```

Comment:
`XWER/H0IkJ8JWFm+n3/lr7hHoNFAX/VpRD+AfpZo2EUgQkQKhrMxHa4aYZPogsvY` (just generated from the AV encrypt)